spec/models/food_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  describe 'Testng food model'
+  subject { Food.new(name: 'Carrot Cake', measurement_unit: 'kg', price: 100) }
+  before { subject.save }
+
+  it 'name should be between 3 - 10 characters' do
+    subject.name = 'this should be false because it is more than 10'
+    expect(subject).to_not be_valid
+  end
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'price should not be nil' do
+    subject.price = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'price should not be less than 0' do
+    subject.price = -4
+    expect(subject).to_not be_valid
+  end
+
+  it 'measurment should be kg' do
+    expect(subject.measurement_unit).to eq 'kg'

```

Comment:
- [ ] Great job adding unit tests but you should also add the integration tests as well using `capybara` in order to complete the requirements 
<img width="420" alt="Screenshot 2022-12-16 at 11 07 13" src="https://user-images.githubusercontent.com/61790587/208083454-4ed3e588-4723-48d7-9384-5ddbfff92617.png">